### ISSUE TITLE
Website unreachable on linux based mobile devices due to cloudflare

### ISSUE THREAD

**Issue by boroli** [Trustroots/trustroots#2639]
**Describe the bug**
I'm running debian mobile with phosh on my pinephone. And since the trustroots website is protected by cloudflare, I cannot access it anymore with any browser installed. And as such also any WebApp doesn't work anymore.

**To Reproduce**
1. Go to www.trustroots.org
2. Cloudflare is popping on Firefox
3. I can click to verfy, and nothing happens.
4. If I refresh the site, I just get back and need to click again to confirm that I'm a human.

If I use angelfish (installed via flatpak, not apt) I get a 403 error, once forwarded to cloudflare.

With Gnome Web, I get an error that my browser is too old and not supported. Which I think is not correct, as it's the newest version in debian testing.

**Expected behavior**
I understand that the website needs protection, but some way to get access to it would be nice.

**Smartphone (please complete the following information):**
 - Device: PinePhone
 - OS: Mobian (Debian Mobile with Phosh)
 - Browser Firefox, Angelfish, Gnome Web
 - Version: latest updates

**Comment by robokow** [comment]
Hi @boroli this should work for you again. Can you confirm? Thanks for getting in touch about it.

**Comment by boroli** [comment]
Yes, I can access the website now again (also with the WebApp). Thanks for the quick fix.
